src/daemon/start_mon.sh

```diff
@@ -95,7 +97,7 @@ function start_mon {
       CEPH_PUBLIC_NETWORK=$(get_network "${nic_more_traffic}" "${NETWORK_AUTO_DETECT}")
       ip_version=${NETWORK_AUTO_DETECT}
     else # Means -eq 1
-      MON_IP="[$(get_ip "${nic_more_traffic}" 6)]"
+      MON_IP="$(get_ip "${nic_more_traffic}" 6)"
```

Comment:
same here

Reply:
In current implementation this extra bracket causes double bracket, `get_ip` already returns the IPv6 with brackets (if it returns anything for IPv6 at all). Adding extra bracket all the time goes against the empty variable check on line 101/103.